### DISCUSSION TITLE
How to access Camilla settings

### DISCUSSION THREAD

**Discussion by lepicsaintloup** [rern/rAudio#1701] [General]
I know this should be obvious but I can't find how to access the Camilla GUI. I turn Camilla on in settings page but then I come to a dead end. Is it just me?

**Comment by rern** [comment]
Click/Tap the `?` icon at the top of the page or each section to show descriptions.

**Comment by lepicsaintloup** [comment]
If I do that then I get the following but not any access: 
![image](https://github.com/rern/rAudio/assets/43662321/918022f2-de34-4131-a648-85272e62fd76)

**Reply by rern** [reply]
Settings > Features `DSP` <<< Here

**Comment by lepicsaintloup** [comment]
Aah! Thank you.
